microsite/pages/en/docs/features/software-catalog/index.js

```diff
@@ -0,0 +1,12 @@
+const React = require('react');
+const Redirect = require('../../../../../core/Redirect.js');
```

Comment:
I have no clue how this thing works. The mentioned namespace has no `Redirect.js` https://github.com/backstage/backstage/tree/master/microsite/core

I used the existing URL redirect for /docs as an example https://github.com/backstage/backstage/blob/master/microsite/pages/en/docs.js

👻 modules

Reply:
Holy dots, batman

Reply:
No sane human can trace back those dots to the original folder. Only machines can.